pkg/virt-operator/kubevirt.go

```diff
@@ -835,13 +835,6 @@ func (c *KubeVirtController) loadInstallStrategy(kv *v1.KubeVirt, loadObservedVe
 	}
 
 	config := operatorutil.GetTargetConfigFromKV(kv)
-	if loadObservedVersion {
```

Comment:
Since you removed it, `loadObservedVersion` bool can be removed from the function's signature.

Reply:
+1 good catch

Reply:
@danielBelenky @davidvossel Thank you. Done.